tools/github/print-client-release-issue.ts

```diff
@@ -0,0 +1,44 @@
+import yargs from "yargs";
+
+async function main() {
+  const argv = yargs(process.argv.slice(2))
+    .usage("Usage: npm run ts-node github/generate-gh-issue-client-release.ts [args]")
+    .version("1.0.0")
+    .options({
+      from: {
+        type: "string",
+        describe: "previous client version",
+        required: true,
+      },
+      to: {
+        type: "string",
+        describe: "next client version",
+        required: true,
+      },
+    })
+    .demandOption(["from", "to"])
+    .help().argv;
+
+  const previousVersion = argv.from;
+  const newVersion = argv.to;
+
```

Comment:
Needs a check to verify `previousVersion` and `newVersion` is not null

Reply:
This is already checked by `yargs` as these 2 fields are set to `required: true`.

Reply:
is it required or demandOption that should be used ?
Anyway, looks good

Reply:
After checking in the doc of `yargs` it is `demandOption` which must be used, but we use it so it's good, we could remove the required field, a cleaning I propose to do in a future PR especially since it concerns several scripts.